gemhardware/managers/src/common/amc/AMCMonitor.cc

```diff
@@ -301,7 +301,7 @@ void gem::hw::glib::GLIBMonitor::setupHwMonitoring()
                  GEMUpdateType::HW32, "hex");
 
   addMonitorableSet("Trigger Status", "HWMonitoring");
-  for (uint8_t oh = 0; oh < p_glib->getSupportedOptoHybrids(); ++oh) {
+  for (uint8_t oh = 0; oh < p_amc->getSupportedOptoHybrids(); ++oh) {
```

Comment:
i mean, it was *working* during the PR MWGR tests, but as i commented in that FIXME, (and it's really **those** types of changes i don't like to merge in) that the lack of information about what is failing would lead me to just leave unchanged those bits of code and fix the bugs during the subsequent work (especially since this function *should* work...)